roles/splunk/handlers/main.yml

```diff
@@ -73,7 +73,9 @@
   when: not start_splunk_handler_fired
 
 - name: restart redhat auditd service
-  command: service auditd condrestart
+  shell: |
+    service auditd stop
```

Comment:
why do you need to stop and start the service here?

Reply:
Yes, That's according to the RedHat documentation
https://access.redhat.com/solutions/2664811